contract/src/@types/zoe-contract-facet.d.ts

```diff
@@ -0,0 +1,8 @@
+type CopyRecord<T> = import('@endo/pass-style').CopyRecord<T>;
+
+// export type {ContractMeta} from '@agoric/zoe/src/contractFacet/types-ambient';
```

Comment:
why doesn't the export from work?

Reply:
because `@agoric/zoe` from agoric-upgrade-14 doesn't include that file.